core/tests/webdriverio_utils/ExplorationEditorSettingsTab.js

```diff
@@ -72,18 +74,18 @@ var ExplorationEditorSettingsTab = function() {
   };
 
   this.expectAvailableFirstStatesToBe = async function(names) {
-    await waitFor.visibilityOf(
-      initialStateSelect,
-      'Initial state select takes too long to be visible.');
-    await waitFor.numberOfElementsToBe(
-      initialStateSelector, 'Options', 3, '<option>',);
-    var options = await initialStateSelect.$$('<option>')
-      .map(async function(elem) {
-        await waitFor.visibilityOf(
-          elem,
-          'option element taking too long to appear');
-        return await elem.getText();
-      });
+    await waitFor.presenceOf(
+      initialStateSelect, 'Initial state select takes too long to be visible.');
+    await action.click('State Dropdown element', initialStateSelect);
+
+    var options = await $$(
```

Comment:
When I migrated this test I was waiting for 3 options to be present using `waitFor.numberOfElementsToBe`, because it use to take some time for the options to be present inside DOM. Are the options available at that instant without waiting as you have removed the waiting function?

Reply:
I have updated the code in the setting-tab.html file. now this component is only rendered when options are present, so we don't need the waitFor.numberOfElementsToBe.